tools/packaging/kata-deploy/Dockerfile

```diff
@@ -3,8 +3,9 @@
 # SPDX-License-Identifier: Apache-2.0
 
 # Specify alternative base image, e.g. clefos for s390x
-ARG IMAGE
-FROM ${IMAGE:-registry.centos.org/centos}:7
+ARG IMG_NAME=registry.centos.org/centos
```

Comment:
Just curious, why not keep the name `IMAGE` as it was and add `IMAGE_TAG`, since this is apparently what you are doing?

Reply:
Also, I'm trying to assess if there is some important corner case of shifting from shell default value to ARG default value… The new approach seems definitely cleaner.

Reply:
I am basing the work on what's already part of the CCv0 branch.  I can change it, no problem.